### PR TITLE
Feat/Provide common params with `.yml` file

### PR DESCRIPTION
**Pull request by notimetoname**
https://github.com/nspcc-dev/neofs-dev-env/pull/241's adaptation.

### CODE REVIEW

File: services/http_gate/cfg/config.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+logger:
+  level: debug # Log level
```
[the remark this answers is not in the document]
Comment (notimetoname) [reply services/http_gate/cfg/config.yml:2]: well, retuned but dont see any reasons for it to be `debug`

File: services/storage/cfg/config.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# Morph section
+morph:
+  dial_timeout: 30s  # Timeout for side chain NEO RPC client connection
+  rpc_endpoint:  # Side chain NEO RPC endpoints
+    - address: ws://morph-chain:30333/ws
+      priority: 1
+
+# Common storage node settings
+node:
+  attribute_0: "User-Agent:NeoFS/0.34"
```
[the remark this answers is not in the document]
Comment (notimetoname) [reply services/storage/cfg/config.yml:26]: returned `0.33`

> Why do we care about this attribute at all?

no idea. actualize/drop in sep commit?

[the remark this answers is not in the document]
Comment (notimetoname) [reply services/storage/cfg/config.yml]: i even have a better [example](https://github.com/nspcc-dev/neofs-node/blob/e158b1e4cc9f2c81b20049dba8b0a141baf72b10/config/testnet/config.yml#L24)

[the remark this answers is not in the document]
Comment (roman-khimov) [reply services/storage/cfg/config.yml]: It makes zero sense to me. Either the node should set it automatically (yeah with some possibility to override) or it shouldn't exist. See nspcc-dev/neofs-contract#299 also.

File: services/http_gate/cfg/config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (roman-khimov) [reply services/http_gate/cfg/config.yml]: At least this should be changed separately then.

File: services/storage/cfg/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (notimetoname) [reply services/storage/cfg/config.yml]: dropped

**Review by roman-khimov — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Review by vvarg229 — APPROVED** [review]
I checked - the changes do not break tests on GitHub:

https://github.com/vvarg229/neofs-node/actions/runs/5269697804/jobs/9528191791

**Pull request merged**